styles/modules/modals/_loading.scss

```diff
@@ -14,6 +14,8 @@
 
     .loadingText {
       @include center;
+      width: 100%;
```

Comment:
They're not. They were necessary for a while when the page structure was different, I forgot to take them out when I changed it.